### ISSUE TITLE
can you elaborate more on your approach to generate synthetic questions

### ISSUE THREAD

**Issue by Sandy4321** [Pleias/Pleias-RAG-Library#2]
great code thanks
only can you elaborate more on your approach to generate synthetic questions 

as you wrote in https://github.com/Pleias/Pleias-RAG-Library/blob/main/Pleias_RAG_Model_Family.pdf
"Figure 6 describes a simplified workflow of the overall process. It starts with the creation of a
synthetic query using a “back-translation” approach (Sennrich et al., 2016): we extract one excerpt at
random and use a fine-tuned language model (Gemma 3 12B) to generate a question this text could
answer."

which exactly fine-tuned language model (Gemma 3 12B) to generate a question model you used?
can  you share this model pls

**Comment by Pclanglais** [comment]
Hi. We trained our own internal model for this, using the base Gemma. Typically for synthetic generation we found better results with structured input rather than classic instruct prompting (so seed text + some typology of query we describe briefly in the paper).
We might release it at some point or, more likely, another model based on the same principle.
